new-player/src/main/java/net/newpipe/newplayer/ui/common/utils.kt

```diff
@@ -55,6 +56,38 @@ import net.newpipe.newplayer.R
 import net.newpipe.newplayer.uiModel.EmbeddedUiConfig
 import java.util.Locale
 
+/** Get the [Activity] from local context. Assumes the activity exists!
+ * @return the activity
+ * @throws NullPointerException if there is no Activity
+ */
```

Comment:
Consider adding @hide here, so this function does not show up on the KDoc documentation.